addons/hr_holidays/models/hr_employee.py

```diff
@@ -168,3 +168,21 @@ def write(self, values):
             allocations = self.env['hr.leave.allocation'].sudo().search([('state', 'in', ['draft', 'confirm']), ('employee_id', 'in', self.ids)])
             allocations.write(hr_vals)
         return res
+
+    def action_time_off_analysis(self):
+        domain = [('holiday_type', '=', 'employee')]
+
+        if self.env.context.get('active_ids'):
+            domain.append(('employee_id', 'in', self.env.context.get('active_ids')))
```

Comment:
Please use expression.AND for this, and don't forget to use a fallback value in case active_ids are not in the context for any reason

```suggestion
            expression.AND([
                domain, 
                ['employee_id', 'in', self.env.context.get('active_ids', [])]
            ])
```